registries/manifest.hocon

```diff
@@ -145,9 +145,24 @@
     "LinkedInJobSeekerSupportNetwork.hocon": true,
     # Experimental and research
     "agent_network_designer.hocon": true,
-    "agent_network_editor.hocon": true,
-    "agent_network_instructions_editor.hocon": true,
-    "agent_network_query_generator.hocon": true,
+    "agent_network_editor.hocon": {
+        # This is a support network for agent_network_designer
+        # We want it served up, but not listed as a public agent network
+        "serve": true,
+        "public": false
+    },
+    "agent_network_instructions_editor.hocon": {
+        # This is a support network for agent_network_designer
+        # We want it served up, but not listed as a public agent network
+        "serve": true,
+        "public": false
+    },
+    "agent_network_query_generator.hocon": { 
+        # This is a support network for agent_network_designer
+        # We want it served up, but not listed as a public agent network
+        "serve": true,
+        "public": false
+    },
```

Comment:
There might be more than this to hide as implementation details for other networks, but for the moment, these are the ones I care about (plus it advertises the feature).